19/README.md

```diff
@@ -0,0 +1,93 @@
+Another major issue with the current implementation is that a new dynamically added validator (`V_i`) cannot be propagated to any new dynamically added validators added after `V_i` i.e. consider the following scenario,
+
+
+- Given a 4 node network wherein the `genesis.json` file contains each of the four nodes `{N1,N2,N3,N4}` a validator.
+- When a new node `N5` is to be dynamically added, each node executes `upsert-validator` on their respective nodes (including `N5`) and adds `N5`.
+- When another new node `N6` needs to be added then each of the nodes in the network `{N1,N2,N3,N4,N5}` execute `upsert-validator` to add the new node `N6`.
+- At this point the node `N6` won't see `N5` as a validator because the `genesis.json` would only contain `{N1,N2,N3,N4}` as validators.
+- Since `N5` was added dynamically and `upsert-validator` only mutates the local validator set of a given node it implies that in order for the `N6` to see `N5` as a validator it would have to execute `upsert-validator N5_PUB_KEY N5_POWER` on its own node at the exact block height (when syncing with the network) when `N5` was previously added by the network.
+
+
+From the above description it is evident that propagating dynamically added validators is a major hassle with huge possibility of errors.
+
+
+## Technical details
+
+To solve the aforementioned issue, following change to the behaviour of `upsert-validator` is being proposed. We use the transaction election process (TEP) proposed in [BEP 18](https://github.com/bigchaindb/BEPs/pull/44) to automate and synchronize the operation on the validator set.
+
+Consider a network of 4 nodes `{A,B,C,D}`. If a node `A` wishes to add a new node `E` to the network then following steps should be followed,
+
+1. Node `A` executes
+
+```
+$ bigchaindb upsert-validator E_PUBKEY E_POWER E_NODE_ID --private-key /home/user/.tendermint/config/priv_validator.json
+<election_id>
+```
+
+The above command `POST`s a [`VALIDATOR_ELECTION`][spec_validator_election] transaction and returns the `election_id`.
+
+NOTE:
+- The `VALIDATOR_ELECTION` transaction is signed with the private key of the validator proposing the election. The private key is generated by Tendermint, and stored in the `priv_validator.json` in the Tendermint's `$TMHOME` directory (usually located at `~/.tendermint/`).
+
+- `E_POWER` should be such that `0 < E_POWER < TOTAL_POWER/3` where `TOTAL_POWER` denotes the total voting power of the current network.
+
```

Comment:
Makes sense to put it here that there has to be an output for every validator in the network.

Reply:
A general description of the election process was missing for I addressed that and your concern here
https://github.com/bigchaindb/BEPs/pull/45/files#diff-8e56cd5a0d784d78d68efd53fa940687R69